### DISCUSSION TITLE
Run two nginx-proxy instances on one host

### DISCUSSION THREAD

**Discussion by toastie89** [nginx-proxy/nginx-proxy#2022] [Q&A]
Hi! I would like two run two nginx-proxy instances on the same docker host, e.g. one on port 80 and another on port 81.
Is there an option to tell the exposing containers which of both instances should be used?

Background: I would like to expose one proxy to the public internet and have the other available in the local network only.

Thanks in advance for your help and ideas!

**Comment by tkw1536** [comment]
Don't think there is such an option at the moment.
